user_interaction.py

```diff
@@ -8,65 +8,58 @@ class MessageScope(Enum):
     PASSIVE = auto()
 
 
-prepared_messages = {
-    MessageScope.GLOBAL: [],
-    MessageScope.ACTIVE: [],
-    MessageScope.PASSIVE: []
-}
+class UserInteraction():
+    def __init__(self):
 
+        self.prepared_messages = {
```

Comment:
Please, make prepared_messages protected. It SHOULD NOT be used outside UserInteraction class
self.prepared_messages -> self._prepared_messages

Reply:
Done

Reply:
Fixed all variables